compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1972,4 +1972,11 @@ object messages {
     val msg = hl"Cannot extend ${"sealed"} $pclazz in a different source file"
     val explanation = "A sealed class or trait can only be extended in the same file as its declaration"
   }
+
+  case class MissingCompanionForStatic(member: Symbol)(implicit ctx: Context) extends Message(MissingCompanionForStaticID) {
+    val msg = hl"${member.owner} does not have a companion class"
+    val kind = "Syntax"
+    val explanation =
+      hl"object that contains @static members must have a companion class."
```

Comment:
```scala
hl"Object that contain ${"@static"} members must have a companion class."
```

Reply:
I propose a slight change:
`hl"An object that contains ${"@static"} members must have a companion class."`

Reply:
Sure